src/components/restclient-component.tsx

```diff
@@ -0,0 +1,29 @@
+import { Heading } from './ui/typography';
+import { useTranslations } from 'next-intl';
+import RestClient from './rest-client/rest-client';
+import { ReactNode } from 'react';
+
+export default function RestclientComponent({
```

Comment:
```suggestion
export default function RestClientComponent({
```
Maintain a consistent naming style